modules/loader/src/test/java/com/opengamma/strata/loader/csv/TradeCsvLoaderTest.java

```diff
@@ -817,6 +819,71 @@ private SwapTrade expectedSwap6() {
         .build();
   }
 
+  private SwapTrade expectedSwap7() {
+    Swap expectedSwap = Swap.builder()
+        .legs(
+            KnownAmountSwapLeg.builder()
+                .payReceive(PAY)
+                .accrualSchedule(PeriodicSchedule.builder()
+                    .startDate(date(2017, 8, 8))
+                    .endDate(date(2022, 8, 8))
+                    .frequency(Frequency.P3M)
+                    .businessDayAdjustment(BusinessDayAdjustment.of(PRECEDING, GBLO.combinedWith(USNY)))
+                    .stubConvention(StubConvention.SMART_INITIAL)
+                    .build())
+                .paymentSchedule(PaymentSchedule.builder()
+                    .paymentFrequency(Frequency.P3M)
+                    .paymentDateOffset(DaysAdjustment.NONE)
+                    .build())
+                .amount(ValueSchedule.of(2_000_000, ValueStep.of(LocalDate.of(2018, 6, 1), ValueAdjustment.ofReplace(2_500_000))))
+                .currency(GBP)
+            .build(),
+
+            //ValueSchedule.of(
```

Comment:
No commented out code

Reply:
Sorry forgot to remove that - removed now